CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 23.1.2 [#671](https://github.com/openfisca/openfisca-core/pull/671)
+
+- Publish both Python 2 and Python 3 version on Pypi
```

Comment:
Use the long, proper version changelog: `Technical improvement` etc. As a reuser not knowing about CI, this is mystifying, help me figure out I don't have to feel concerned!

Reply:
I don't think we ever agreed on a taxonomy of changesets in Core. What would not be a technical improvement?

Reply:
Happy to give more details about the consequences of the change though.

Reply:
Oh sorry, I reviewed too quickly 😞